src/app/App.js

```diff
@@ -1,13 +1,35 @@
-import React from 'react'
-import { Route, Routes } from 'react-router-dom'
+import React, { useEffect, useState } from "react";
+import { BrowserRouter as Router, Routes, Route} from 'react-router-dom'
 import Home from '../pages/Home/Home'
+import Login from '../components/Login/Login';
+import Signup from '../components/Signup/Signup';
+import Profile from '../components/Profile/Profile';
+import Statistics from '../components/Statistics/Statistics';
```

Comment:
Can you import it like `import Statistics from '../components/Statistics';` please?

Reply:
Fixed